src/mixins/Themeable.ts

```diff
@@ -237,3 +237,5 @@ export function ThemeableMixin<T extends WidgetConstructor>(base: T): Constructo
 		}
 	};
 }
+
+export default ThemeablMixin;
```

Comment:
Ha that's a crappy (a bug in the existing code that I have spelt the interface incorrectly so it let me export that instead of the function!)